lib/services/ibm_devops.rb

```diff
@@ -1,8 +1,9 @@
 class Service::IBMDevOpsServices < Service::HttpPost
-  string :ibm_id
+  title 'IBM Bluemix DevOps Services'
+  string :ibm_login
```

Comment:
Any existing services will still send whatever they had configured. Any new services will use the new `ibm_login` fields. It does become quite confusing, unfortunately.